Vagrantfile

```diff
@@ -0,0 +1,73 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  # This can be changed to anything else, but keep in mind the states
+  # are only written with 14.04.1 in mind.
```

Comment:
@pnorman Not as of yet; the trusty64 box has yet to be updated to 14.04.2. This isn't much of a problem, as .2 mainly consists of kernel and hardware support updates (beyond other typical package updates).